### PR TITLE
Fixed over-flow bug on widgets with longer names

### LINKED ISSUE #3630: Width of the widget handle should adjust according to the name
### What is the expected behaviour?

Width of the widget handle should adjust according the length of widget name and the delete button next to it.

### What is the current behaviour?

Currently, widgets like password input, vertical divider and many others with longer widget names have widget name and delete button going out of bound (Please go through all the widgets with longer names)

### How to reproduce the issue?

1. drag widgets on the canvas
2. hover through their handle

### Screenshots or Screencast

https://user-images.githubusercontent.com/25361949/179498666-e81e0eda-64f0-42ef-9a4c-e0c625aa529b.mov

### PR DESCRIPTION
**Pull request by UtkarshDixit-py**
Increased default widget width by 2px.(Only for widgets having longer names than the widget width itself)

fixes #3630

### CODE REVIEW

**Comment by withshubh** [comment]
Hey @UtkarshDixit-py 
Can you please check all the widgets on ToolJet cloud and fix whichever widget is having this issue as I can see you have fixed the issue with two widgets in this PR.
The PR is supposed to fix this issue completely so that it doesn't happen with any widget in the future.

**Comment by UtkarshDixit-py** [comment]
@stepinfwd Does this need any other changes?

File: frontend/src/Editor/WidgetManager/widgetConfig.js [hunk 1]
```diff
@@ -1978,7 +1978,7 @@ export const widgets = [
     description: 'Show the progress using circular progressbar',
     component: 'CircularProgressBar',
     defaultSize: {
-      width: 7,
+      width: 8,
```
Comment (stepinfwd) [inline frontend/src/Editor/WidgetManager/widgetConfig.js:1981]: revert this line , CircularProgressBar size doesn't need any change

**Comment by stepinfwd** [comment]
> @stepinfwd Does this need any other changes?

mentioned one change rest looks good @UtkarshDixit-py

**Comment by UtkarshDixit-py** [comment]
Hey @stepinfwd i made the mentioned change

**Review by stepinfwd — APPROVED** [review]
LGTM @Navaneeth-pk 

https://user-images.githubusercontent.com/31006028/183377806-65a2ac21-ded8-48c9-83d2-228fbb23337c.mov

**Comment by stepinfwd** [comment]
> Hey @stepinfwd i made the mentioned change

There are lint issues can you fix that too @UtkarshDixit-py

**Comment by UtkarshDixit-py** [comment]
> > Hey @stepinfwd i made the mentioned change
> 
> There are lint issues can you fix that too @UtkarshDixit-py

I don't know how to debug this linting issue

**Comment by stepinfwd** [comment]
> > > Hey @stepinfwd i made the mentioned change
> > 
> > 
> > There are lint issues can you fix that too @UtkarshDixit-py
> 
> I don't know how to debug this linting issue

cc: @arpitnath

**Review by arpitnath — CHANGES_REQUESTED** [review]
On renaming any widget, the app freezes!
Will remove the tag `do not merge` after this is fixed.
cc: @Navaneeth-pk

**Comment by arpitnath** [comment]
@UtkarshDixit-py,
Simply determine the config-handler's width with respect to the length of the component's name.


File: **/frontend/src/Editor/ConfigHandle.jsx**


https://user-images.githubusercontent.com/67645175/183990961-59554a28-e1b3-4f6a-bd9c-02e43b95a806.mov

**Comment by UtkarshDixit-py** [comment]
> @UtkarshDixit-py, Simply determine the config-handler's width with respect to the length of the component's name.
> 
> File: **/frontend/src/Editor/ConfigHandle.jsx**
> 
>  Screen.Recording.2022-08-11.at.12.01.01.AM.mov

Okay sure

**Comment by withshubh** [comment]
Hey @UtkarshDixit-py 👋 
Are you working on it?

**Comment by arpitnath** [comment]
Closing this PR, as the issue is re-assign to someone else.
cc: @withshubh

**Pull request closed without merging**